### PR TITLE
Fix inability to pip install

### LINKED ISSUE #10: Installation does not work.
"git clone https://github.com/Omyyyy/pycom.git
cd pycom/src
pip install -r requirements.txt
pip install -e ."
The src folder does not contain a requiements.txt file. So i did go to the parent dir, where the requirements txt is and installed it.
Then i did pip install -e . 
But then the next problem. When i try to start it now with pycom in the console it gives an error ending with "ModuleNotFoundError: No module named 'src'

### PR DESCRIPTION
**Pull request by 0dminnimda**
Fixes #10

There are a whole array of downsides to adding the dot in the package name and even more if it's `.py`.
Also it's not heplfull to name modules like the package, so `pycom.py` should be replaced with the `__main__.py`, this way we don't even need the console_scripts.
So let's not use those names, it makes things harder for everybody ^-^

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
-from setuptools import setup
+from setuptools import find_packages, setup
+
 setup(
     name='pycom.py',
     version='1.0.0',
-    entry_points={
-        'console_scripts': [
-            'pycom=src.pycom.pycom:main'
```
Comment (riebecj) [inline setup.py:7]: Blowing away the entrypoint script defeats the purpose

File: src/pycom/__main__.py [hunk 2]
```diff
@@ -9,9 +9,7 @@
 
 from colorama import Fore
 
-import compiler
-import errors
-import tokenise
+from . import compiler, errors, tokenise
```
Comment (riebecj) [inline src/pycom/__main__.py:12]: It has been updated to use absolute importing with namespaces to provide greater flexibility as the package grows.

**Comment by riebecj** [comment]
This should now be fixed with commit 08803a4b45f8d01ca53badd244e6fe85ff277586

File: src/pycom/__main__.py (hunk 2 above)
Comment (0dminnimda) [reply src/pycom/__main__.py:12]: relative imports are as flexible, the only restriction is you cannot run the files directly, you need to import the package instead

**Pull request closed without merging**